packages/netlify-cms-lib-auth/src/implicit-oauth.js

```diff
@@ -42,12 +42,22 @@ export default class ImplicitAuthenticator {
     authURL.searchParams.set('response_type', 'token');
     authURL.searchParams.set('scope', options.scope);
 
-    const state = JSON.stringify({ auth_type: 'implicit', nonce: createNonce() });
```

Comment:
`auth_type: 'implicit'` is important as it provides compatibility with Netlify Identity Widget when using implicit auth

Reply:
Hm, think I introduced that while rebasing. Fixed.